packages/workflows-engine-redis/src/services/workflows-module.ts

```diff
@@ -0,0 +1,199 @@
+import {
```

Comment:
suggestion: DTOs for method arguments would be a neat addition to make it more clean